pkg/managerclient/restore.go

```diff
@@ -6,21 +6,25 @@ package managerclient
 const (
 	RestoreStageInit          = "INIT"
 	RestoreStageDropViews     = "DROP_VIEWS"
+	StageDisableCompaction    = "DISABLE_COMPACTION"
```

Comment:
Please follow the same pattern by either removing `Restore` prefix from all the stages, or adding prefix `Restore` to the new stages.